pages/posts/page/[page_number].tsx

```diff
@@ -0,0 +1,87 @@
+import fs from 'node:fs';
+import type { GetStaticPaths, GetStaticProps } from 'next';
+import type { DualPost } from '../../../lib/types';
+import PostPreview from '../../../components/PostPreview';
+import { getAllPosts } from '../../../lib/getAllPosts';
+import { podcastXml } from '../../../lib/podcast';
+import PageWrapper from '../../../components/PageWrapper';
+import Paginator from '../../../components/Paginator';
+import { paginate } from '../../../lib/helpers';
+
+export const getStaticProps: GetStaticProps = async (context) => {
+  const allPosts = await getAllPosts();
+  const whichPage = Number(context.params?.page_number);
+  const thisPagePosts = paginate(
+    allPosts.sort(
+      (a, b) => new Date(b.createdAt).getTime() - new Date(a.createdAt).getTime(),
+    ),
+    whichPage,
+    8,
+  );
+  const enPosts = allPosts.map((dual) => dual.en);
+  const esPosts = allPosts.map((dual) => dual.es);
+  fs.writeFileSync(`./public/podcast.en.rss`, podcastXml(enPosts));
+  fs.writeFileSync(`./public/podcast.es.rss`, podcastXml(esPosts));
+
+  return {
+    props: {
+      posts: thisPagePosts,
+      pageCount: Math.ceil(allPosts.length / 8),
+      pageNum: whichPage,
+    },
+  };
+};
+
+export const getStaticPaths: GetStaticPaths = async () => {
+  const allDualPosts = await getAllPosts();
+  const numberOfPages = Math.ceil(allDualPosts.length / 8);
+  const pages = new Array(numberOfPages).fill(0).map((_, index) => index + 1);
+
+  const paths = pages.map((page) => ({
+    params: { page_number: String(page) },
+  }));
+
+  return { paths, fallback: false };
+};
+
+interface Props {
+  posts: DualPost[];
+  pageNum: number;
+  pageCount: number;
+}
+
+const Posts: React.FC<Props> = ({ posts, pageNum, pageCount }) => {
+  return (
+    <PageWrapper
+      page="/posts"
+      withChrome
+      language="en"
+      redirectTo={`/publicaciones/pagina/${pageNum}`}
+      title="Posts | The Ancient Path"
```

Comment:
ideally, (at least on non-page 1) the title should be different, maybe like `Posts (page 2) | Ancient Path`

Reply:
well, i guess i was thinking of the _seo_ title tag, not sure if that's what this prop is.

Reply:
but also, i feel moderately strongly like the right thing to do would be to also have the visible page heading have some sort of indication that you're on Page 2 or 3 as well. do you know what i mean?